src/repositories/ColorPaletteRepository.spec.ts

```diff
@@ -100,24 +100,24 @@ describe('ColorPaletteRepositoryTest', () => {
 
     test('should map create color palette from frontify to api correctly', () => {
         expect(
-            mapFrontifyColorPaletteCreateToColorPaletteApiCreate(FrontifyColorPaletteCreateDummy.with(), PROJECT_ID),
+            mapFrontifyColorPaletteCreateToColorPaletteApiCreate(ColorPaletteCreateDummy.with(), PROJECT_ID),
         ).toEqual(ColorPaletteApiCreateDummy.with(PROJECT_ID));
     });
 
     test('should map patch type correctly from frontify type to api type', () => {
-        expect(mapFrontifyColorPalettePatchToColorPaletteApiPatch(FrontifyColorPalettePatchDummy.with())).toEqual(
+        expect(mapFrontifyColorPalettePatchToColorPaletteApiPatch(ColorPalettePatchDummy.with())).toEqual(
             ColorPaletteApiPatchDummy.with(),
         );
     });
 
     test('should map correctly from api type to frontify type with colors', () => {
         expect(mapColorPaletteApiToFrontifyColorPalette(ColorPaletteApiDummy.with(COLOR_PALETTE_ID))).toEqual(
-            FrontifyColorPaletteDummy.with(COLOR_PALETTE_ID),
+            ColorPaletteDummy.with(COLOR_PALETTE_ID),
         );
     });
 
     test('should map correctly from api type to frontify type without colors', () => {
-        const frontifyColorPalette = FrontifyColorPaletteDummy.with(COLOR_PALETTE_ID);
+        const frontifyColorPalette = ColorPaletteDummy.with(COLOR_PALETTE_ID);
```

Comment:
We should update the constant name as well:
```suggestion
        const colorPalette = ColorPaletteDummy.with(COLOR_PALETTE_ID);
```